src/test/kotlin/com/github/michaelbull/retry/policy/BackoffTest.kt

```diff
@@ -72,6 +73,16 @@ class BackoffTest {
             assertEquals(expectedDelay, instruction.delayMillis)
         }
     }
+    @Test
+    fun `full jitter random bounds`() {
+        runBlockingTest(RetryRandom(mockk<Random>(relaxed = true).apply {
+            every { nextLong(any()) } answers { 0 }
+            every { nextLong(any(), any()) } answers { firstArg()  }
+        })) {
```

Comment:
Can we split this up? All of the mocking can be done in a separate variable, then we can simplify pass the `RetryRandom` to `runBlockingTest`